sap-3-tier-marketplace-image-converged-md/azuredeploy.json

```diff
@@ -0,0 +1,996 @@
+      },
+      "Windows Server 2012 R2 Datacenter": {
+        "sku": "2012-R2-Datacenter",
+        "offer": "WindowsServer",
+        "publisher": "MicrosoftWindowsServer",
+        "version": "latest",
+        "OSType": "Windows",
+        "Plan": {},
+        "UsePlan": false
+      },
+      "Windows Server 2016 Datacenter": {
+        "sku": "2016-Datacenter",
+        "offer": "WindowsServer",
+        "publisher": "MicrosoftWindowsServer",
+        "version": "latest",
+        "OSType": "Windows",
+        "Plan": {},
+        "UsePlan": false
+      },
+      "SLES 12": {
+        "sku": "12-SP1",
+        "offer": "SLES-SAP",
+        "publisher": "SUSE",
+        "version": "latest",
+        "OSType": "Linux",
+        "Plan": {},
+        "UsePlan": false
+      },
+      "SLES 12 BYOS": {
+        "sku": "12-SP1",
+        "offer": "SLES-SAP-BYOS",
+        "publisher": "SUSE",
+        "OSType": "Linux",
+        "version": "latest",
+        "Plan": {},
+        "UsePlan": false
+      },
+      "RHEL 7.2": {
+        "sku": "7.2",
+        "offer": "RHEL",
+        "publisher": "RedHat",
+        "version": "latest",
+        "OSType": "Linux",
+        "Plan": {},
+        "UsePlan": false
+      },
+      "Oracle Linux 7.2": {
+        "sku": "7.2",
+        "offer": "Oracle-Linux",
+        "publisher": "Oracle",
+        "version": "latest",
+        "OSType": "Linux",
+        "Plan": {},
+        "UsePlan": false
+      }
+    },
+    "imageSku": "[variables('images')[parameters('osType')].sku]",
+    "imagePublisher": "[variables('images')[parameters('osType')].publisher]",
+    "imageOffer": "[variables('images')[parameters('osType')].offer]",
+    "imageVersion": "[variables('images')[parameters('osType')].version]",
+    "internalOSType": "[variables('images')[parameters('osType')].OSType]",
+    "csExtension": {
+      "Windows": {
+        "script": "[concat(parameters('_artifactsLocation'), 'noop.ps1')]",
+        "scriptCall": "powershell.exe -ExecutionPolicy bypass -File noop.ps1"
+      },
+      "Linux": {
+        "script": "[concat(parameters('_artifactsLocation'), 'noop.sh')]",
+        "scriptCall": "sh noop.sh"
+      }
+    },
+    "vmSizes": {
+      "Demo": {
+        "Not HA": {
+          "clserversize": "Standard_DS12_v2",
+          "clservercount": 1,
+          "clserverdisks": [
+            {
+              "size": 128
+            }
+          ],
+          "diserversize": "Standard_DS2_v2",
+          "diservercount": 1
+        },
+        "HA": {
+          "clserversize": "Standard_DS12_v2",
+          "clservercount": 2,
+          "clserverdisks": [
+            {
+              "size": 128
+            }
+          ],
+          "diserversize": "Standard_DS2_v2",
+          "diservercount": 2
+        }
+      },
+      "Small < 30.000 SAPS": {
+        "Not HA": {
+          "clserversize": "Standard_DS13_v2",
+          "clservercount": 1,
+          "clserverdisks": [

```

Comment:
not sure if that would make it less confusing :) there are also some VM sizes that have different disk sizes so not all of them could be converted to copy loops